src/axom/sidre/core/MFEMSidreDataCollection.hpp

```diff
@@ -20,6 +20,7 @@ namespace sidre
 {
 /** @brief Data collection with Sidre routines following the Conduit mesh
     blueprint specification. */
+// clang-format off
```

Comment:
Thanks for fixing this!

Does `clang-format` alter our comments?  I thought we disabled that option.